MAUI/DataForm/migration.md

```diff
@@ -0,0 +1,961 @@
+</tr>
+
+<tr>
+<td>{{'[ValidationMode](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.ValidationMode.html)'| markdownify }}</td>
+<td>{{'[DataFormValidationMode]'| markdownify }}</td>
+<td>Defines the constants for ValidationMode that specifies when the user input should be validated for data object.</td>
+</tr>
+</table>
+
+## Events
+
+<table>
+<tr>
+<th>Xamarin SfDataForm</th>
+<th>.NET MAUI SfDataForm</th>
+<th>Description</th></tr>
+
+<tr>
+<td>{{'[AutoGeneratingDataFormItem](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.SfDataForm.html)'| markdownify }}</td>
+<td>{{'[GenerateDataFormItem]'| markdownify }}</td>
+<td>Occurs during the generation of data form item in the data form when AutoGenerateItems is true.</td>
+</tr>
+
+<tr>
+<td>{{'[Validating](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.SfDataForm.html#Syncfusion_XForms_DataForm_SfDataForm_Validating)'| markdownify }}</td>
+<td>{{'[ValidateProperty]'| markdownify }}</td>
+<td>Occurs when SfDataForm validates the user input.</td>
+</tr>
+
+<tr>
+<td>{{'[ValidationCompleted](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.SfDataForm.html#Syncfusion_XForms_DataForm_SfDataForm_ValidationCompleted)'| markdownify }}</td>
+<td>{{'[ValidateForm]'| markdownify }}</td>
+<td>Occurs when DataFormValidationMode is Manual to get the invalid DataFormItem’s details.</td>
+</tr>
+</table>
+
+## Methods
+
+<table>
+<tr>
+<th>Xamarin SfDataForm</th>
+<th>.NET MAUI SfDataForm</th>
+<th>Description</th></tr>
+
+<tr>
+<td>{{'[RegisterEditor](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.SfDataForm.html#Syncfusion_XForms_DataForm_SfDataForm_RegisterEditor_System_String_Syncfusion_XForms_DataForm_Editors_DataFormEditorBase_)'| markdownify }}</td>
+<td>{{'[RegisterEditor]'| markdownify }}</td>
+<td>Mthod to register DataFormEditor.</td>
+</tr>
+
+<tr>
+<td>{{'[Validate](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.SfDataForm.html#Syncfusion_XForms_DataForm_SfDataForm_Validate().html)'| markdownify }}</td>
+<td>{{'[Validate]'| markdownify }}</td>
+<td>Validates all the fields of DataFormViewItem</td>
+</tr>
+
+<tr>
+<td>{{'[Commit](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.SfDataForm.html#Syncfusion_XForms_DataForm_SfDataForm_Commit().html)'| markdownify }}</td>
+<td>{{'[Commit]'| markdownify }}</td>
+<td>Method to commit all properties underlying data object from the editor view.</td>
+</tr>
+
+<tr>
+<td>{{'[UpdateEditor](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.SfDataForm.html#Syncfusion_XForms_DataForm_SfDataForm_UpdateEditor_System_String_)'| markdownify }}</td>
+<td>{{'[UpdateEditor]'| markdownify }}</td>
+<td>Updates the value of the specific property in corresponding editor.</td>
+</tr>
+</table>
+
+## Upcoming Features
+
+## Known issues in .NET MAUI DataForm
+
```

Comment:
## Known issues in .NET MAUI DataForm

- [MacCatalyst] - TimePicker popup is not working.
- [MacCatalyst, iOS] - Multiline editor placeholder text font size not changing based on editor font size.
- [MacCatalyst, iOS] – Date and Time picker text styles are not changing.
- [MacCatalyst, iOS] – Date picker minimum and maximum display dates are not working.
-  [MacCatalyst, iOS] - Text style not applied to radio button.
- [Android, iOS, MacCatalyst] - In RTL mode, the multiline text editor text is not showing from the right.
- [Windows] - Picker editor text style is not changing.
- [iOS] - Maximum input length is not working in Text and Password editors.
- [Windows] - Time picker format not changing.
- [MacCatalyst, iOS] - RTL FlowDirection is not working for Date and Time pickers.
- [MacCatalyst, iOS] - In RTL FlowDirection, the ScrollView scroll bar shows at the right.
- [Android] - in RTL mode Radio button's content does not arrange from the right.
- RTL flow direction not working for ComboBox and AutoComplete editors.
- IsReadOnly not working for AutoComplete editor.

Reply:
Added known issues in .NET MAUI DataForm